src/tss2-tcti/tctildr.c

```diff
@@ -361,12 +361,13 @@ Tss2_TctiLdr_GetInfo (const char *name,
     rc = copy_info (info_lib, info_tmp);
     if (rc != TSS2_RC_SUCCESS) {
         free (info_tmp);
+        info_tmp = NULL;
         goto out;
     }
     info_tmp->init = NULL;
 out:
     tctildr_finalize_data (&data);
-    *info = info_tmp;
+    *info = info_tmp ? info_tmp : NULL;
```

Comment:
This does not make too much sense, does it ?
If NULL then NULL
If not NULL then the value ? ;-)
I think the fix in L364 is sufficient...

Reply:
I agree that this check is redundant, but that's the only way to make the coverity happy.

Reply:
No, L364 is what fixes coverity. L370 does nothing, IMHO

Reply:
updated